### DISCUSSION TITLE
Is there a getting started guide?

### DISCUSSION THREAD

**Discussion by brandiqa** [remix-run/remix#2628] [Q&A] [answered]
We need a dedicated page for showing how to setup a blank Remix project without any stacks.

**Comment by sergiodxa** [comment, marked as answer]
You can do `npx create-remix@latest` and instead of choosing a Stack. Here:

<img width="458" alt="Screen Shot 2022-04-04 at 12 46 17" src="https://user-images.githubusercontent.com/1312018/161601567-b880bb66-58fb-43df-85f8-f44ff7e04db6.png">

The `Just the basics` options gives you a blank Remix app

**Reply by brandiqa** [reply]
I am writing an e-commerce tutorial and I would like to link to a getting started guide in the event the project creation process changes in the future.

Also I've noted that the remix dependency is missing when you create the basic project.

**Reply by sergiodxa** [reply]
> I am writing an e-commerce tutorial and I would like to link to a getting started guide in the event the project creation process changes in the future

The CLI option to use Just the basics should never be removed. The idea is that the CLI helps to get a basic template (just the minimum required code) or use Stacks which are more complex ones.

> Also I've noted that the remix dependency is missing when you create the basic project

They're working on removing that, so you import directly from the `@remix-run/*` packages

**Comment by brandiqa** [comment]
Thanks for the clarification @sergiodxa. I have now updated the tutorial project code using the latest `@remix-run/* ` packages.

I do hope there are plans for documenting the use of `npx create-remix@latest` since the docs at it's current state doesn't reflect this information.

**Reply by sergiodxa** [reply]
The first page of the docs mention the CLI command https://remix.run/docs/en/v1#getting-started

**Comment by brandiqa** [comment]
🤦‍♂️Aah....I totally missed that. When I click on the Getting Started button on the home page it takes me to the [Quick start Tutorial Blog](https://remix.run/docs/en/v1/tutorials/blog). Didn't realize the Read the Docs takes you to the actual Getting Started page.

May I suggest at least adding a **Getting Started link** in the `Side Bar` under `Pages` section?
